code/modules/cards/deck/tarot.dm

```diff
@@ -0,0 +1,50 @@
+//These cards certainly won't tell the future, but you can play some nice games with them.
+/obj/item/toy/cards/deck/tarot
+	name = "tarot game deck"
+	desc = "A full 78 card game deck of tarot cards. Complete with 4 suites of 14 cards, and a full suite of trump cards."
+	icon = 'icons/obj/toy.dmi'
+	icon_state = "deck_tarot_full"
+	deckstyle = "tarot"
+	is_standard_deck = FALSE
+
+/obj/item/toy/cards/deck/tarot/Initialize(mapload)
+	. = ..()
+	for(var/suit in list("Hearts", "Pikes", "Clovers", "Tiles"))
+		for(var/i in 1 to 10)
+			cards += new /obj/item/toy/singlecard(mapload, "[i] of [suit]", src)
+		for(var/person in list("Valet", "Chevalier", "Dame", "Roi"))
+			cards += new /obj/item/toy/singlecard(mapload, "[person] of [suit]", src)
+	for(var/trump in list("The Magician", "The High Priestess", "The Empress", "The Emperor", "The Hierophant", "The Lover", "The Chariot", "Justice", "The Hermit", "The Wheel of Fortune", "Strength", "The Hanged Man", "Death", "Temperance", "The Devil", "The Tower", "The Star", "The Moon", "The Sun", "Judgement", "The World", "The Fool"))
+		cards += new /obj/item/toy/singlecard(mapload, trump, src)
+
+/obj/item/toy/cards/deck/tarot/draw(mob/user)
+	. = ..()
+	var/obj/item/toy/singlecard/C = .
+	var/matrix/M = matrix()
```

Comment:
😠

Reply:
Tarot code is busted right now.  Planning to add the card rotation into the rotation component I refactored in a recent PR that got merged.

Reply:
no i just mean the var name

Reply:
Applied with https://github.com/tgstation/tgstation/pull/64200/commits/5c9d5f72b0ebb9b7eb030370071c3bda2564fc75

One letter variable names are normally frowned upon but if it's in a part of the code where it has a short lifespan then there isn't an issue.  That logic is why you see `i` in a for loop for a lot of programing languages.  These are supposed to be throw away vars for a small chunk of code.